### PR TITLE
hotfix for profiles

### PR DESCRIPTION
**Pull request by DuckySoLucky**

### CODE REVIEW

File: src/lib.js [hunk 1]
```diff
@@ -3267,93 +3230,56 @@ export async function getProfile(
 
       allSkyBlockProfiles.push(doc);
 
-      lastCachedSave = Math.max(lastCachedSave, doc.members[paramPlayer].last_save || 0);
+      lastCachedSave = Math.max(lastCachedSave, Date.now() || 0);
```
Comment (dukio) [inline src/lib.js]: This will always be `Date.now()`, right?

File: src/constants/hotm.js [hunk 2]
```diff
@@ -759,7 +759,7 @@ class PeakOfTheMountain extends Node {
     const output = [];
 
     for (let tier = 1; tier <= level; tier++) {
-      for (const [reward, qty] of Object.entries(rewards.potm[tier])) {
+      for (const [reward, qty] of Object.entries(rewards.hotm[tier])) {
```
Comment (dukio) [inline src/constants/hotm.js]: this `potm` was correct, if it creates other errors those needs to be fixed

**Review by dukio — CHANGES_REQUESTED** [review]
didn't test, reviewed code on github and left a couple comments

File: src/lib.js (hunk 1 above)
Comment (DuckySoLucky) [reply src/lib.js]: Yeah, it's useless will remove tomorrow

File: src/constants/hotm.js (hunk 2 above)
Comment (DuckySoLucky) [reply src/constants/hotm.js]: Yeah I'm aware as I said on discord they have updated the hotm tree. This fixes error so it's kinda a fix for it as well. Will update it tomorrow

Comment (dukio) [reply src/constants/hotm.js]: image

Oh shit missed this in patch notes... welp found why then

File: src/lib.js (hunk 1 above)
Comment (DuckySoLucky) [reply src/lib.js]: Solved, didn't remove found better way

File: src/constants/hotm.js (hunk 2 above)
Comment (DuckySoLucky) [reply src/constants/hotm.js]: Solved

**Pull request merged**